tools/emscripten.py

```diff
@@ -328,7 +328,10 @@ def emscript(in_wasm, out_wasm, outfile_js, js_syms, finalize=True, base_metadat
   else:
     # Skip finalize and only extract the metadata.
     if in_wasm != out_wasm:
-      shutil.copy(in_wasm, out_wasm)
+      if settings.MEMORY64:
+        building.run_wasm_opt(in_wasm, out_wasm, ['--table64-lowering'])
```

Comment:
We seem to do all the other `-lowering` passes in `tools/link.py`, including `table64-lowering`. Perhaps a comment could explain why this extra place is needed?

Reply:
Good point.  I moved this code and added a comment.